conda/core/initialize.py

```diff
@@ -1187,13 +1189,13 @@ def init_long_path(target_path):
     # win10, build 14352 was the first preview release that supported this
     if int(win_ver) >= 10 and int(win_rev) >= 14352:
         prev_value, value_type = _read_windows_registry(target_path)
-        if prev_value != "1":
+        if prev_value != 1:
```

Comment:
Given the ambiguity here, whey not cast it just to be safe? `int(prev_value)`

Reply:
Actually, the safer choice is probably `if str(prev_value) != "1":` in case `prev_value` is `None`

Reply:
Hmm... reading a registry value of type REG_DWORD should always return an int. But I understand that we wouldn't want a traceback in bad scenarios anyway.